diff -uNr qtemu.orig/helpwindow.cpp qtemu/helpwindow.cpp
--- qtemu.orig/helpwindow.cpp	2007-03-19 13:56:21.000000000 +0100
+++ qtemu/helpwindow.cpp	2007-06-20 22:47:25.000000000 +0200
@@ -22,6 +22,7 @@
 ****************************************************************************/
 
 #include <QtGui>
+#include "config.h"
 
 #include "helpwindow.h"
 
@@ -59,10 +60,15 @@
 {
     QSettings settings("QtEmu", "QtEmu");
     QString locale = settings.value("language", QString(QLocale::system().name())).toString();
+    QString version (VERSION);
     if (locale != "en")
     { //check first if there is a language specific help available
         //check for case when qtemu executable is in same dir (windows)
-        QUrl testUrl("help/" + locale + "/main.htm");
+        QUrl testUrl("/usr/share/qtemu-" + version + "/help/" + locale + "/main.htm");
+        if (QFile::exists(testUrl.toString()))
+            return testUrl;
+    
+        testUrl = QUrl("help/" + locale + "/main.htm");
         if (QFile::exists(testUrl.toString()))
             return testUrl;
     
@@ -77,8 +83,12 @@
             return testUrl;
     }
 
+    QUrl testUrl("/usr/share/qtemu-" + version + "/help/main.htm");
+    if (QFile::exists(testUrl.toString()))
+	    return testUrl;
+    
     //check for case when qtemu executable is in same dir (windows)
-    QUrl testUrl("help/main.htm");
+    testUrl = QUrl("help/main.htm");
     if (QFile::exists(testUrl.toString()))
         return testUrl;

