--- qtemu-1.0.5/helpwindow.cpp~	2011-09-14 07:54:10.000000000 +0300
+++ qtemu-1.0.5/helpwindow.cpp	2011-09-14 07:54:52.911356540 +0300
@@ -22,6 +22,7 @@
 ****************************************************************************/
 
 #include "helpwindow.h"
+#include "config.h"
 
 #include <QFile>
 #include <QCoreApplication>
@@ -59,10 +60,15 @@
 {
     QSettings settings("QtEmu", "QtEmu");
     QString locale = settings.value("language", QString(QLocale::system().name())).toString();
+    QString version (VERSION);
     if (locale != "en")
     { //check first if there is a language specific help available
         //check for case when qtemu executable is in same dir (windows)
-        QUrl testUrl("help/" + locale + "/main.htm");
+        QUrl testUrl("/usr/share/qtemu-" + version + "/help/" + locale + "/main.htm");
+        if (QFile::exists(testUrl.toString()))
+            return testUrl;
+    
+        testUrl = QUrl("help/" + locale + "/main.htm");
         if (QFile::exists(testUrl.toString()))
             return testUrl;
     
@@ -77,8 +83,12 @@
             return testUrl;
     }
 
+    QUrl testUrl("/usr/share/qtemu-" + version + "/help/main.htm");
+    if (QFile::exists(testUrl.toString()))
+	    return testUrl;
+    
     //check for case when qtemu executable is in same dir (windows)
-    QUrl testUrl("help/main.htm");
+    testUrl = QUrl("help/main.htm");
     if (QFile::exists(testUrl.toString()))
         return testUrl;

